domain-services/fiscal-event-service/src/main/resources/db/migration/main/V20220728121459__DDL_fiscal_event_create_ddl.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE eg_ifix_fiscal_event (
+    id varchar,
+    tenantid varchar NOT NULL,
```

Comment:
Use camelCase here. Technically, it will not make any difference as SQL is case insensitive, but it will improve readability. 
Camel case is used In most of the persister config files and SQL migration files.

Reply:
Won't do